tests/test_reserve_now.py

```diff
@@ -0,0 +1,134 @@
+from paho.mqtt import client as mqtt_client
```

Comment:
I'm not sure about including this just yet, things especially with python based testing will change quite a bit in the near future. I would rather remove this for now and introduce it when all the pieces are in place